src/watcher.js

```diff
@@ -99,13 +101,23 @@ class Watcher extends SafeEventEmitter {
                         this.emit('load.chat');
                         break;
                     case 'directory.following.index':
-                        this.emit('load.following');
+                        this.emit('load.following.overview');
                         // Switching between tabs in following page
                         if (lastRoute.substr(0, 19) === 'directory.following') break;
                         this.emit('load.directory.following');
                         break;
+                    case 'directory.channels.all':
```

Comment:
these all seem to do the same thing, why are they not all grouped together as a simplistic `load.directory`